lib/snapshot/mem_fs.go

```diff
@@ -659,9 +659,6 @@ func (fs *MemFS) untarDirectory(path string, header *tar.Header) error {
 // untarSymlink creates the symlink specified by header at path.
 func (fs *MemFS) untarSymlink(path string, header *tar.Header) error {
 	target := header.Linkname
-	if filepath.IsAbs(header.Linkname) {
-		target = filepath.Join(fs.tree.src, target)
```

Comment:
In production / builds, fs.tree.src is always `"/"`, so this is a no-op. However, when we extract the rootfs, this line breaks symlinks.